CameraXBasic/gradle.properties

```diff
@@ -1,16 +1,17 @@
-# Project-wide Gradle settings.
-# IDE (e.g. Android Studio) users:
-# Gradle settings configured through the IDE *will override*
-# any settings specified in this file.
-# For more details on how to configure your build environment visit
+## For more details on how to configure your build environment visit
 # http://www.gradle.org/docs/current/userguide/build_environment.html
+#
 # Specifies the JVM arguments used for the daemon process.
 # The setting is particularly useful for tweaking memory settings.
-org.gradle.jvmargs=-Xmx1536m
-android.enableJetifier=true
-android.useAndroidX=true
-android.enableUnitTestBinaryResources=true
+# Default value: -Xmx1024m -XX:MaxPermSize=256m
+# org.gradle.jvmargs=-Xmx2048m -XX:MaxPermSize=512m -XX:+HeapDumpOnOutOfMemoryError -Dfile.encoding=UTF-8
+#
 # When configured, Gradle will run in incubating parallel mode.
 # This option should only be used with decoupled projects. More details, visit
 # http://www.gradle.org/docs/current/userguide/multi_project_builds.html#sec:decoupled_projects
 # org.gradle.parallel=true
+#Tue Dec 24 15:36:07 PKT 2019
```

Comment:
I wonder how it is related to fixed issue :-) I think it should be in another merge request with updated gradle build tools version ;-)

Reply:
Sir what I implemented is better way to solve in Kotlin and if your intention is to only resolve the issue then close this pull request.

Reply:
:ok_hand: